test/e2e/test-fully-automated.sh

```diff
@@ -36,6 +36,16 @@ cleanup() {
     return
   fi
   pushd "${MIGRATOR_ROOT}"
+    echo "===== initializer logs"
+    kubectl logs --namespace=kube-system job/initializer || true
```

Comment:
This test doesn't launch the initializer, right?

Reply:
@caesarxuchao Done